css/start-use-css/properties-in-css/table.html

```diff
@@ -9,7 +9,51 @@
 </head>
 <body>
     <div>
-        
+        <h1 class="wrapper">Высота в таблице</h1>
+        <p class="first-p">Наследование позволяет задавать значения некоторых свойств единожды, 
+            определяя их для родителей верхнего уровня.</p>
+        <h2 class="wrapper">Ширина в таблице</h2>
+        <p class="second-p">Самым низким приоритетом обладает стиль браузера — оформление,
+             которое по умолчанию применяется к элементам веб-страницы браузером.</p>
+    </div>
+    <div>         
+        <table border="5" widht="400" height="200">
```

Comment:
Don't use properties for style description on an any tags. This property need use in CSS and plug their through class

Reply:
corrected